spec/merchant_spec.rb

```diff
@@ -0,0 +1,9 @@
+require './lib/merchant'
+
+RSpec.describe Merchant do
+  it 'exists' do
+    m = Merchant.new({:id => 5, :name => "Turing School"})
+
+    expect(m).to be_instance_of(Merchant)
+  end
```

Comment:
Need a test for the time now portion of the initialize to ensure those are being set correctly.